setup.py

```diff
@@ -149,7 +149,7 @@ def run(self):
       test_requires=test_requires,
       packages=['pyop2', 'pyop2.codegen'],
       package_data={
-          'pyop2': ['assets/*', '*.h', '*.pxd', '*.pyx']},
+          'pyop2': ['assets/*', '*.h', '*.pxd', '*.pyx', '*.c']},
```

Comment:
Does this pick up the C files, I think it needs to be `'codegen/c/*.c'` (If I read https://docs.python.org/3/distutils/setupscript.html#installing-package-data correctly).